pkg/controllers/placement/controller.go

```diff
@@ -204,6 +212,12 @@ func (r *Reconciler) handleUpdate(ctx context.Context, placementObj fleetv1beta1
 		return ctrl.Result{}, err
 	}
 
+	// Sync ClusterResourcePlacementStatus object if StatusReportingScope is NamespaceAccessible
+	if err := r.syncClusterResourcePlacementStatus(ctx, placementObj); err != nil {
```

Comment:
Is it better to update the crp's status first then crps? If we update crps first, and then updating crp status fails, crps lastUpdateTime can be earlier than crp's status sync time and from user's perspective they may consider crps as not up-to-date?

Reply:
https://github.com/kubefleet-dev/kubefleet/pull/225#discussion_r2305475639